packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-backup

```diff
@@ -1,36 +1,185 @@
 #!/bin/bash
```

Comment:
I realize this isn't a recent change, but the shebang lines should all be `#!/usr/bin/env bash`